src/diffusers/models/embeddings.py

```diff
@@ -563,7 +565,7 @@ def __call__(self, x):
 
 
 class PositionNet(nn.Module):
-    def __init__(self, positive_len, out_dim, fourier_freqs=8):
+    def __init__(self, positive_len, out_dim, feature_type, fourier_freqs=8):
```

Comment:
```suggestion
    def __init__(self, positive_len, out_dim, feature_type = "text-only", fourier_freqs=8):
```

for backwards compatibility